frontend/src/components/molecules/QuickSearchInput/QuickSearchInput.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  color: ${({ theme }) => theme.color.grey300};
+  background-color: ${({ theme }) => theme.color.grey100};
+  border-radius: 5px;
+  padding: 8px 16px;
+  font-family: ${({ theme }) => theme.fontFamily};
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+const IconBox = styled.div`
```

Comment:
Remove IconBox. Try this:
![image](https://user-images.githubusercontent.com/33421875/117874072-d865e500-b2a0-11eb-8845-16a850d883b1.png)
:)